Todo/Todo.iOS/Todo.iOS.csproj

```diff
@@ -172,9 +172,6 @@
       <Name>Todo</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup>
-    <InterfaceDefinition Include="Resources\LaunchScreen.storyboard" />
```

Comment:
a launchscreen storyboard tells the operating system that the app can adjust to multiple screen resolutions - it should not be deleted

with launch screen:
<img width="273" alt="screen shot 2018-03-15 at 2 07 06 pm" src="https://user-images.githubusercontent.com/199373/37491236-6545fb60-285a-11e8-80d6-69f77b835fec.png">

without launchscreen:
<img width="273" alt="screen shot 2018-03-15 at 2 07 23 pm" src="https://user-images.githubusercontent.com/199373/37491253-6f567e22-285a-11e8-9af8-d14d80af7408.png">

Reply:
The LaunchScreen was not configured in `info.plist`. I'll update this to re-add it, add a reuse id to fix the warning, and configure Info.plist.